### DISCUSSION TITLE
Automatic hyperparameter selection

### DISCUSSION THREAD

**Discussion by slavakx** [autogluon/autogluon#2432] [Q&A] [answered]
Hi, 

in this [TDS article](https://towardsdatascience.com/autogluon-deep-learning-automl-5cdb4e2388ec)  it was mentioned 
"The hyperparameters of each task are automatically selected using advanced tuning algorithms such as Bayesian Optimization, Hyperband, and Reinforcement Learning. With AutoGluon, you don’t have to have any familiarity with the underlying models, as all hyperparameters will be automatically tuned within default ranges that are known to perform well for the particular task and model."

Can you please explain, how the hyperparameters are being automatically selected. 
How is it different from hyperparameter tuning? 
Is the difference only in the preset parameters known to perform well rather then by providing my own parameter ranges for each algorithm?

Thanks

**Comment by Innixma** [comment, marked as answer]
We generally just use preset hyperparameters that generally perform well rather than doing any hyperparameter tuning at present (AutoGluon v0.1+). This is very similar in strength to extensive hyperparameter tuning, at a fraction of the training time.
